# HG changeset patch
# User sbr <sebastien@brissaud.name>
# Date 1375201511 -7200
#      Tue Jul 30 18:25:11 2013 +0200
# Node ID da4bb7c8114ea81bd1d682375f8610d3aec90bfc
# Parent  f467cd5ef652eda5e105bf59a14936e48598126d
-Fix [FS#5679] (r20055): Make 'PLUS' an usable hotkey by mapping it to the '+' key instead of to the '+' char.

diff --git a/src/hotkeys.cpp b/src/hotkeys.cpp
--- a/src/hotkeys.cpp
+++ b/src/hotkeys.cpp
@@ -54,7 +54,7 @@ static const KeycodeNames _keycode_to_na
 	{"F11", WKC_F11},
 	{"F12", WKC_F12},
 	{"PAUSE", WKC_PAUSE},
-	{"PLUS", (WindowKeyCodes)'+'},
+	{"PLUS", WKC_PLUS},
 	{"COMMA", WKC_COMMA},
 	{"NUM_PLUS", WKC_NUM_PLUS},
 	{"NUM_MINUS", WKC_NUM_MINUS},
